.mergify.yml

```diff
@@ -0,0 +1,61 @@
+pull_request_rules:
+  - name: automatic merge for main when CI passes and approved
+    conditions:
+      - "#approved-reviews-by>=1"
+      # Unfortunately, we cannot just say "all CI must pass".
```

Comment:
Would it be possible to just consolidate all the tests into a single test so that we only need one `check-success`? I think that might make the CI itself faster too, as it could avoid recompiling av1an for each test. Perhaps we could also circumvent the need to download the test videos, and just generate very short (like <1s) 64x64 black screen videos to use as test clips.